cmd/organization.go

```diff
@@ -0,0 +1,250 @@
+		if err != nil {
+			return err
+		}
+		if err := requiredInputCheck("Organization name", organizationName); err != nil {
+			return err
+		}
+		if err != nil {
+			return err
+		}
+		organizationFriendlyName, err := cmd.Flags().GetString("friendly-name")
+		if err != nil {
+			return err
+		}
+		organizationDescription, err := cmd.Flags().GetString("description")
+		if err != nil {
+			return err
+		}
+		organizationQuotaProject, err := cmd.Flags().GetUint("project-quota")
+		if err != nil {
+			return err
+		}
+		organizationQuotaGroup, err := cmd.Flags().GetUint("group-quota")
+		if err != nil {
+			return err
+		}
+		organizationQuotaNotification, err := cmd.Flags().GetUint("notification-quota")
+		if err != nil {
+			return err
+		}
+		organizationQuotaEnvironment, err := cmd.Flags().GetUint("environment-quota")
+		if err != nil {
+			return err
+		}
+		organizationQuotaRoute, err := cmd.Flags().GetUint("route-quota")
+		if err != nil {
+			return err
+		}
+
+		current := lagoonCLIConfig.Current
+		token := lagoonCLIConfig.Lagoons[current].Token
+		lc := lclient.New(
+			lagoonCLIConfig.Lagoons[current].GraphQL,
+			lagoonCLIVersion,
+			&token,
+			debug)
+
+		organization, err := l.GetOrganizationByName(context.TODO(), organizationName, lc)
+		organizationInput := s.UpdateOrganizationPatchInput{
+			Description:       nullStrCheck(organizationDescription),
+			FriendlyName:      nullStrCheck(organizationFriendlyName),
+			QuotaProject:      nullUintCheck(organizationQuotaProject),
+			QuotaGroup:        nullUintCheck(organizationQuotaGroup),
+			QuotaNotification: nullUintCheck(organizationQuotaNotification),
+			QuotaEnvironment:  nullUintCheck(organizationQuotaEnvironment),
+			QuotaRoute:        nullUintCheck(organizationQuotaRoute),
+		}
+		result, err := l.UpdateOrganization(context.TODO(), organization.ID, organizationInput, lc)
+		handleError(err)
+
+		resultData := output.Result{
+			Result: "success",
+			ResultData: map[string]interface{}{
+				"Organization Name": result.Name,
+			},
+		}
+		output.RenderResult(resultData, outputOptions)
+		return nil
+	},
+}
+
+func init() {
+	addOrganizationCmd.AddCommand(addOrgCmd)
+	addOrganizationCmd.AddCommand(addGroupToOrganizationCmd)
+	addOrganizationCmd.AddCommand(addProjectToOrganizationCmd)
+	addOrganizationCmd.AddCommand(addDeployTargetToOrganizationCmd)
+	addOrganizationCmd.AddCommand(addUserToOrganizationCmd)
+
+	deleteOrganizationCmd.AddCommand(deleteOrgCmd)
+	deleteOrganizationCmd.AddCommand(RemoveDeployTargetFromOrganizationCmd)
+	deleteOrganizationCmd.AddCommand(RemoveProjectFromOrganizationCmd)
+	deleteOrganizationCmd.AddCommand(RemoveUserFromOrganization)
+
+	addOrgCmd.Flags().StringP("name", "O", "", "Name of the organization")
+	addOrgCmd.Flags().String("friendly-name", "", "Friendly name of the organization")
+	addOrgCmd.Flags().String("description", "", "Description of the organization")
+	addOrgCmd.Flags().Uint("project-quota", 0, "Project quota for the organization")
```

Comment:
All the quotas accept `-1` for `unlimited` this doesn't currently work

Reply:
You'll probably need to update machinery too from from `uint`

Reply:
Good catch, now accounts for `-1`